code/modules/projectiles/guns/ballistic/pistol.dm

```diff
@@ -97,3 +97,17 @@
 		to_chat(user, "<span class='notice'>..and falls into view. Whew, that was a close one.</span>")
 		user.dropItemToGround(src)
 
+/obj/item/gun/ballistic/automatic/liberatorpistol
+	name = "Makeshift Pistol"
+	desc = "Harkening back to the prehistoric Liberator pistol design, this small makeshift pistol fires 10mm bullets."
+	icon_state = "liberatorpistol"
+	item_state = "liberatorpistol"
```

Comment:
can you not call it liberatorpistol, give it a name that makes sense to a reader who might be looking through DMI files